### PR TITLE
Update docs to clarify that all actions require a target

### LINKED ISSUE #456: Update docs to clarify that all actions require a target
We've had users try to say just "crown" or "square wrap" without a target and wonder why it won't parse.  We could support these commands, but worry that they would pollute the command space when you can just say eg "crown this" instead.  For the time being, we'll

- improve the docs to make it clearer that these actions without targets aren't possible
- modify the cheatsheet so that every action has a `T` after it for consistency

### PR DESCRIPTION
**Pull request by AndrewDant**
Closes #456
image

### CODE REVIEW

File: docs/user/README.md
Comment (pokey) [inline docs/user/README.md:474]: Does this one still work? I think we removed this form a while back

File: docs/user/README.md
Comment (pokey) [inline docs/user/README.md:322]:
```suggestion
In any cursorless command, the action defines what happens to the specified target. Every command must have a target. For example, deleting the target (`"chuck"`, as in `"chuck air"`) or moving the cursor to select the target (`"take"`, as in `"take air"`).
```

File: cursorless-talon/src/cheatsheet/sections/actions.py
Comment (pokey) [inline cursorless-talon/src/cheatsheet/sections/actions.py:19]: Please use f-string
```suggestion
        f"{key} <T>": value
```

File: cursorless-talon/src/cheatsheet/sections/actions.py
Comment (pokey) [inline cursorless-talon/src/cheatsheet/sections/actions.py:19]: This change is the one that is most controversial. It's ok with me, but there were objections to how much noise it adds to the cheatsheet cc/ @AndreasArvidsson

**Review by pokey — COMMENTED** [review]
Lots of good improvements here; thanks! I left some comments

File: docs/user/README.md
Comment (AndrewDant) [reply docs/user/README.md:474]: extract T as hello world doesn't seem to work for me. Extract T still does

File: cursorless-talon/src/cheatsheet/sections/actions.py
Comment (AndrewDant) [reply cursorless-talon/src/cheatsheet/sections/actions.py:19]: Talked with Tara Roys and a few other users at the cursorless meetup on 4/10 and they all were in favor of adding the T's for clarity.

File: docs/user/README.md
Comment (AndrewDant) [reply docs/user/README.md:474]: Not sure how to verify for certain but extract ___ as ___ doesn't seem to exist.

Comment (AndrewDant) [reply docs/user/README.md:474]: I've verified it by reading the ExtractVariable action file. l'll update the read me

**Comment by pokey** [comment]
Here's my original PR for what it's worth https://github.com/cursorless-dev/cursorless-talon/pull/112

**Review by pokey — DISMISSED** [review]

**Comment by pokey** [comment]
Curious why this one is still marked as draft. What's missing?

**Comment by AndrewDant** [comment]
Because I marked it when I first made the request and forgot to unmark it.

**Comment by pokey** [comment]
> Because I marked it when I first made the request and forgot to unmark it.

Cool that was my theory 😊

**Comment by pokey** [comment]
Mind adding a screenshot to the PR description? Would be good to do a quick visual sanity check before I hit that merge button

**Comment by AndrewDant** [comment]
image

Not sure how to edit the description but here's a screenshot.

File: cursorless-talon/src/cheatsheet/sections/actions.py
Comment (AndreasArvidsson) [reply cursorless-talon/src/cheatsheet/sections/actions.py:19]: If people prefer it that way: go ahead :)

**Comment by AndrewDant** [comment]
image
I think that looks much better actually.

**Review by pokey — APPROVED** [review]

**Pull request merged**
